doc/doc-site/docs/off-chain/dss/changelog.md

```diff
@@ -306,11 +306,6 @@ Stable DSS builds are tracked on the [`dss-stable`] branch with tags like [`dss-
 [#801]: https://github.com/econia-labs/econia/pull/801
 [#803]: https://github.com/econia-labs/econia/pull/803
 [docs site readme]: https://github.com/econia-labs/econia/blob/main/doc/doc-site/README.md
-[dss-v2.1.0-rc.1]: https://github.com/econia-labs/econia/releases/tag/dss-v2.1.0-rc.1
-[dss-v2.2.1-rc.1]: https://github.com/econia-labs/econia/releases/tag/dss-v2.2.1-rc.1
-[dss-v2.3.0-rc.1]: https://github.com/econia-labs/econia/releases/tag/dss-v2.3.0-rc.1
-[dss-v2.4.0-rc.1]: https://github.com/econia-labs/econia/releases/tag/dss-v2.4.0-rc.1
-[dss-v2.5.0-rc.2]: https://github.com/econia-labs/econia/releases/tag/dss-v2.5.0-rc.2
 [processor #19]: https://github.com/econia-labs/aptos-indexer-processors/pull/19
```

Comment:
```suggestion
[dss-v2.1.0-rc.1]: https://github.com/econia-labs/econia/releases/tag/dss-v2.1.0-rc.1
```

- [ ] Leave this line in since it is referenced at the top of the page

Feel fre to resolve this comment once done